cpp/src/slice2cs/Gen.cpp

```diff
@@ -885,15 +886,15 @@ Slice::CsVisitor::getDispatchParams(
     }
     else if (op->hasMarshaledResult())
     {
-        name = fixId(op->name(), DotNet::ICloneable, true);
+        name = fixId(op->name(), DotNet::Object, true);
```

Comment:
I actually kept most of the `DotNet::ICloneable` "collision avoidance" code, since Clone is part of our generated API, even though we no longer implement ICloneable.

I would be in favor of eliminating all this code, like we did it in IceRPC. Ideally, we'd provide a `cs:identifier` metadata to allow the user to pick a name of his choice.